addons/mail/models/mail_template.py

```diff
@@ -54,8 +54,8 @@ def format_tz(env, dt, tz=False, format=False):
         format_date = langs.date_format or '%B-%d-%Y'
         format_time = langs.time_format or '%I-%M %p'
 
-        fdate = pycompat.text_type(ts.strftime(format_date))
-        ftime = pycompat.text_type(ts.strftime(format_time))
+        fdate = str(ts.strftime(format_date))
+        ftime = str(ts.strftime(format_time))
```

Comment:
Can probably remove the str call entirely, I expect the point was mostly to convert the str (bytes) output of P2's strftime to unicode to format it in a string below.

Also could probably remove the `u` prefixes next line.